src/Context/AuthProvider.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { createContext } from 'react'
+import { useContext } from 'react'
+import { useState } from 'react'
+
+const AuthContext = createContext()
+
+const AuthProvider = ({ children }) => {
+    const localStorageToken = JSON.parse(localStorage.getItem("login"));
+    const [token, setToken] = useState(localStorageToken?.token);
+    const localStorageUser = JSON.parse(localStorage.getItem("user"));
+    const [user, setUser] = useState(localStorageUser?.user);
```

Comment:
Just a suggestion, instead of creating multiple useState variables either you can create a reducer or create an object inside a single useState.